.claude/rules/governance/versioning-migration.md

```
@@ -0,0 +1,113 @@
+| Removed | After deprecated | Error on read |
+
+### Deprecation Warning
+
+```python
+if version < MIN_SUPPORTED_VERSION:
+    warn(f"Receipt version {version} is deprecated. Migrate to {CURRENT_VERSION}.")
+```
+
+## Anti-Patterns
+
+### No Migration Path
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)